packages/account/src/Components/trading-assessment/trading-assessment-form.jsx

```diff
@@ -0,0 +1,194 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Formik, Form } from 'formik';
+import { Button, Modal, FormSubmitButton, Text, Icon } from '@deriv/components';
+import { isMobile } from '@deriv/shared';
+import { localize, Localize } from '@deriv/translations';
+import TradingAssessmentRadioOption from './trading-assessment-radio-buttons';
+import TradingAssessmentDropdownOption from './trading-assessment-dropdown';
+
+const TradingAssessmentForm = ({
+    assessment_questions,
+    class_name,
+    form_value,
+    onSubmit,
+    onCancel,
+    is_header_navigation,
+}) => {
+    const [is_next_button_enabled, setIsNextButtonEnabled] = React.useState(false);
+    const [current_question_details, setCurrentQuestion] = React.useState({
+        current_question_index: 0,
+        current_question: {},
+    });
+    const [form_data, setFormData] = React.useState({});
+
+    const last_question_index = assessment_questions.length - 1;
+
+    React.useEffect(() => {
+        setCurrentQuestion(prevState => ({
+            ...prevState,
+            current_question: assessment_questions[prevState.current_question_index],
+        }));
+        setFormData(form_value);
+    }, []);
+
+    const displayNextPage = () => {
+        if (form_data.risk_tolerance === 'No') {
+            onSubmit(form_data, true, true);
```

Comment:
Question: why do we need to pass two `true` parameters?

Reply:
Will add a comment explaining the params